fastlane_core/lib/fastlane_core/globals.rb

```diff
@@ -0,0 +1,33 @@
+module FastlaneCore
+  class Globals
+    def self.captured_output
+      unless @captured_output
+        @captured_output = ""
+      end
+      @captured_output
+    end
+
+    class << self
+      attr_writer :captured_output
+      attr_writer :capture_output
+      attr_writer :verbose
+    end
+
+    def self.capture_output?
+      return nil unless @capture_output
+      return true
```

Comment:
there are many places where it is explcitily required to be nil, and `false`  would change the behaviour.